loop-server/manifest.json

```diff
@@ -2,16 +2,30 @@
   "envs" : {
     "stage": {
       "test_types": [ "stack-check", "loadtest" ],
+      "stack_label": "loopsvrstg",
+      "aws_region": "eu-west-1",
       "urls": {
 	"root" : "loop.stage.mozaws.net",
-	"push_config" : "loop.stage.mozaws.net/push-server-config"
+	"push_config" : "loop.stage.mozaws.net/push-server-config",
+	"heartbeat" : "loop.stage.mozaws.net/__heartbeat__"
+      }
+    },
+    "pre-prod": {
+      "test_types": [ "stack-check" ],
+      "aws_region": "eu-west-1",
+      "urls": {
+        "root": "loop.services.mozilla.com",
+	"push_config" : "loop.services.mozilla.com/push-server-config",
+	"heartbeat" : "loop.services.mozilla.com/__heartbeat__"
       }
     },
     "prod": {
       "test_types": [ "stack-check" ],
+      "aws_region": "eu-west-1",
       "urls": {
         "root": "loop.services.mozilla.com",
-	"push_config" : "loop.services.mozilla.com/push-server-config"
+	"push_config" : "loop.services.mozilla.com/push-server-config",
+	"heartbeat" : "loop.services.mozilla.com/__heartbeat__"
```

Comment:
agreed.  i was trying not to refactor too soon before we figured out this was the way we wanted to go with our test manifests, but i think we've more or less got it down now, so i'll go ahead and make that change.